crud/select/type_comparators.lua

```diff
@@ -6,7 +6,7 @@ local UnsupportedCollationError = errors.new_class('UnsupportedCollationError')
 
 local types = {}
 
-local function comparing_nil(cmp)
+local function comparing_nullable(cmp)
```

Comment:
It looks like it's `lt_nullable`, not comparing.

Reply:
But it is not. There are three `lt` implementations using the same `nil` values comparison.

Reply:
Does it work for `eq`, `gt` and so on? If yes - it's really `comparing`. If it works only for `lt` - it's `lt`-specific comparison, and function name should reflect it.

Reply:
Renamed